packages/assets-controllers/src/RatesController/RatesController.ts

```diff
@@ -0,0 +1,186 @@
+import { BaseController } from '@metamask/base-controller';
+import { Mutex } from 'async-mutex';
+
+import { fetchMultiExchangeRate as defaultFetchExchangeRate } from '../crypto-compare-service';
+import type {
+  RatesMessenger,
+  ConversionRates,
+  RatesControllerState,
+  RatesControllerOptions,
+} from './types';
+
+export const name = 'RatesController';
+
+const DEFAULT_CURRENCIES = {
```

Comment:
Done in [bc8e9bc](https://github.com/MetaMask/core/pull/4242/commits/bc8e9bc501c5eb426e095ab21d0a9135c803a703)

Reply:
Awesome, that's better, now I wonder if this should be `export`ed? Since we have a method to update the `.fromCurrencies` field, maybe we could allow the caller to use our `enum` too?

Now, additional question, should `.currencies` be using this `enum` type instead of `string`? Or you wanted to leave it open here?

IMO, that's probably better to go for the `enum` for now, and change it to a `string` later on if need be. WDYT? If you agree, then let's change `DefaultCurrencies` to `Currencies` instead maybe?